### DISCUSSION TITLE
Path for Tabler icons not working

### DISCUSSION THREAD

**Discussion by nitindermohan** [thuliteio/doks#1235] [Q&A] [answered]
Hi. I am trying to include the Tabler icons in the markdown but I get an error "Could not find src".

Following the example in the documentation [here](https://getdoks.org/docs/basics/shortcodes/)

```
{{< callout context="caution" title="Caution" icon="alert-triangle" >}}
If you are not sure you want an awesome docs site, think twice before using [Doks](https://getdoks.org/).
{{< /callout >}}
```
I get 

![Screenshot 2024-05-08 at 12 53 49](https://github.com/gethyas/doks/assets/5736850/a29b0c7f-7c7b-4a53-8916-f7fcc881d235)

I also tried adding inline icons 

```
{{< inline-svg "coffee" >}}
```
But they don't show up as well.

I see that the Tabler dir path in `module.toml` is correct

```
[[mounts]]
  source = "node_modules/@tabler/icons/icons"
  target = "assets/svgs/tabler-icons"
```

I am relatively new to Hugo so I am unsure if I am missing anything.

**Comment by h-enk** [comment, marked as answer]
[Tabler Icons 3.0.0](https://github.com/tabler/tabler-icons/releases/tag/v3.0.0) introduced filled icon variants — and a new folder setup. I will update the docs for this — thanks!

So, now it's:

```md
{{< inline-svg "outline/coffee" >}}
```

Or:

```md
{{< inline-svg "filled/coffee" >}}
```

**Reply by nitindermohan** [reply]
I confirm that this works. Thanks for a quick response and for creating such an awesome theme!!
